terrible.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import sys
+import os
+import time
+name='Yuki'
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpcbzce6lr/terrible.py
+++ b/tmp/tmpcbzce6lr/terrible.py
@@ -2,7 +2,9 @@
 import sys
 import os
 import time
-name='Yuki'
+name = 'Yuki'
+
+
 class Greeter:
     def greet(self):
         print('Hello, world!')
```